### PR TITLE
Added flatten function and changed the __repr__ function.

### PR DESCRIPTION
**Pull request by Rohith04MVK**
Added flatten function and changed the __repr__ function to make the matrix look better.

### CODE REVIEW

**Comment by janaSunrise** [comment]
The flake8 checks are behaving weird. Will it be fine If I use the changes and attribute your email i.e. `007rohitjj04@gmail.com` with the attribution and push it? 😁

**Comment by Rohith04MVK** [comment]
And added a little bit of linting

File: hypemaths/models/matrix.py [hunk 1]
```diff
@@ -58,8 +58,19 @@ def dims(self) -> tuple:
...
         return tuple(self._get_mat_dimension(self.matrix))
 
+    @property
+    def size(self) -> int:
+        '''
+        Returns
+        -------
+        int
+            The integer which is the total number of items in the matrix
+        '''
```
Comment (janaSunrise) [inline hypemaths/models/matrix.py:68]:
```suggestion
    def size(self) -> int:
        """
        Returns
        -------
        int
            The integer which is the total number of items in the matrix
        """
```
Use `"` double quotes for eveness

File: hypemaths/models/matrix.py [hunk 2]
```diff
@@ -569,3 +559,38 @@ def from_vector(cls, vector: "hm.Vector") -> "Matrix":
...
+        list, tuple, Vector
+            A tuple containing the dimensions of the matrix passed.
+
+        Examples
+        --------
+        >>> m = hm.Matrix([[1,2], [3,4]])
+        >>> m.flatten()
+        list([[1, 2, 3, 4]])
+
+        '''
```
Comment (janaSunrise) [inline hypemaths/models/matrix.py:582]:
```suggestion
        """
        Return a flattened version of the matrix.
        All elements of the matrix are placed into a single row.
        Parameters
        ----------
        cls : {'list', 'tuple', 'Vector'}, Optional
            Type the flat matrix should be returned
        Returns
        -------
        list, tuple, Vector
            A tuple containing the dimensions of the matrix passed.

        Examples
        --------
        >>> m = hm.Matrix([[1,2], [3,4]])
        >>> m.flatten()
        list([[1, 2, 3, 4]])

        """
```
Same as above

**Review by janaSunrise — CHANGES_REQUESTED** [review]
One more thing is to keep the docstrings with gaps. It's for the doc generator to parse it and understand it easily. Hope you understand.

**Review by janaSunrise — APPROVED** [review]
Good to go! 👍🏼

**Pull request merged**